### DISCUSSION TITLE
It takes too much time for Wallpaper to load

### DISCUSSION THREAD

**Discussion by cheeckhicks** [prasanthrangan/hyprdots#1245] [Q&A]
When I do Super+Alt+Arrow it takes like 5 seconds for wallpaper to change. When I open wallpaper select menu, they all appear grey. When I want to change theme, it doesn't change. What could be the problem/solution? 
![ed633f1a-ea4d-4db0-ad92-eaef0cfc19d3](https://github.com/prasanthrangan/hyprdots/assets/150370767/855365a0-619f-4d67-99a0-2025bbde6674)

**Comment by kRHYME7** [comment]
It's not cache yet.  did you install the main branch?

**Reply by cheeckhicks** [reply]
I'm a total beginner. Can you explain what it means. I used the installation guide video https://www.youtube.com/watch?v=mb8h1-LB9K0 I guess so.

**Reply by kRHYME7** [reply]
Hey, this current main branch will be updated soon(The video too will be outdatd). So your local configs might be deprecated soon. 

Here's the 
https://github.com/kRHYME7/Hyde-cli 

install ``` hyde-cli-git ```  then run ``` Hyde-install ```  then follow the instructions during installation

Your configs will be backed up so you can retrieve it later.
